service/Abstractions/Pipeline/MimeTypes.cs

```diff
@@ -68,10 +73,12 @@ public class MimeTypesDetection : IMimeTypeDetection
             { FileExtensions.Htm, MimeTypes.Html },
             { FileExtensions.Html, MimeTypes.Html },
             { FileExtensions.WebPageUrl, MimeTypes.WebPageUrl },
-            { FileExtensions.MsWord, MimeTypes.MsWord },
-            { FileExtensions.MsWordX, MimeTypes.MsWord },
-            { FileExtensions.MsPowerPoint, MimeTypes.MsPowerPoint },
-            { FileExtensions.MsExcel, MimeTypes.MsExcel },
+            // { FileExtensions.MsWord, MimeTypes.MsWord }, // TODO: add support for legacy doc files
+            { FileExtensions.MsWordX, MimeTypes.MsWordX },
+            // { FileExtensions.MsPowerPoint, MimeTypes.MsPowerPoint }, // TODO: add support for legacy ppt files
+            { FileExtensions.MsPowerPointX, MimeTypes.MsPowerPointX },
+            // { FileExtensions.MsExcel, MimeTypes.MsExcel }, // TODO: add support for legacy xls files
```

Comment:
no need to comment out the mappings, they are useful when logging warnings/errors